lib/controller.js

```diff
@@ -741,6 +741,35 @@ controller.disableAccessPoint = function(opts) {
   });
 };
 
+controller.getAccessPointInfo = function(opts) {
+  opts.authorized = true;
+  return controller.standardTesselCommand(opts, function(tessel) {
+    return tessel.getAccessPointInfo()
+      .then((ap) => {
+        if (ap.ssid) {
+          logs.info('SSID: ' + ap.ssid);
+
+          if (ap.password || ap.security !== 'none') {
+            logs.info('Password: ' + ap.password);
+          }
+
+          logs.info('Security: ' + ap.security);
+          logs.info('IP Address: ' + ap.ip);
+          logs.info(!Number(ap.disabled) ? 'Enabled' : 'Disabled');
+        } else {
+          logs.info(tessel.name + ' is not configured as an access point (run "t2 ap --help" to learn more)');
+        }
+
+        return controller.closeTesselConnections([tessel]);
```

Comment:
@HipsterBrown You actually don't need to call `closeTesselConnections` because when you called `standardTesselCommand` on line 746 above, it wraps your callback such that it automatically closes Tessel once your Promise resolves.